### ISSUE TITLE
Collapsible accessibility issues

### ISSUE THREAD

**Issue by jkmcnabb** [telus/tds-core#222]
Collapsible components (http://tds.telus.digital/4-Components/Collapsible.html) should do the following in order to be accessible: 
1) be triggered by a button with a label - the example has this
2) the button should have aria-expanded="true"/"false" - the examples have this, so we don't need the aria-live region with the collapsed/expanded text
3) the button should have aria-controls="id" where id points to the section being expanded
And more issues:
4) The "+" icon is very far to the right side of the page, where people using screen magnifiers might not see it. And without it, they won't know "Panel #1" is clickable.
5) NVDA reads "Panel #1 body" when the Panel #1 is collapsed. This seems like a defect.

The above points apply to the Accordion as well.
